perf/WebJobs.Script.Benchmarks/CSharpCompilationBenchmarks.cs

```diff
@@ -1,6 +1,10 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Can we seperate the style fixes from the changes to packages & .props?

Reply:
Let me see if I can get that done. The package props standardized StyleCop, which brought in all these warnings to some packages.

Reply:
Ah those are all git history getting weird when I backed out the other .cs changes. They are all what is _already_ in dev. Not sure why it is reporting as a change.

Reply:
re-merging dev fixed it. The remaining .cs file change is valid, I dropped a package since its only usage was an unused utility method